apps/account-worker/src/worker.config.ts

```diff
@@ -23,8 +23,8 @@ export default registerAs('account-worker', (): IAccountWorkerConfig => {
       label: 'API_BODY_JSON_LIMIT',
       joi: Joi.string().default('1mb'),
     },
-    workerApiPort: {
-      label: 'WORKER_API_PORT',
+    apiPort: {
+      label: 'API_PORT',
```

Comment:
`apiPort` is great for the config variable name. Might want to consider a different name for the env var, though. Maybe keep `'WORKER_API_PORT'`? The reason: when using environment files to provision, we typically have a common env file across the api & worker. ie, 'account-api' and 'account-worker' are both driven by 'account.env', so we want to be able to distinguish between the two ports there.

Other solutions to consider:
* use `WORKER_API_PORT` if it exists; otherwise use `API_PORT`
* Split all the environment files so that each app has its own env file in all scripting & Docker setup; ie 'account-api.env' and 'account-worker.env'